MultiServer.py

```diff
@@ -1549,9 +1590,9 @@ async def process_client_cmd(ctx: Context, client: Client, args: dict):
             create_as_hint: int = int(args.get("create_as_hint", 0))
             hints = []
             for location in args["locations"]:
-                if type(location) is not int or location not in lookup_any_location_id_to_name:
+                if type(location) is not int:
                     await ctx.send_msgs(client,
-                                        [{'cmd': 'InvalidPacket', "type": "arguments", "text": 'LocationScouts',
+                                        [{'cmd': 'InvalidPacket', "type": "argument", "text": 'LocationScouts',
```

Comment:
the `network protocol.md` says arguments was correct

Reply:
that was a bleed-through from something else, will fix.